frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupRulesForm/groupRulesInputs/ruleFieldValues/fieldValueInput/fieldValueInput.component.tsx

```diff
@@ -24,7 +24,7 @@ import { get } from 'lodash';
 import { ListboxComponent } from './listboxComponent/listboxComponent.component';
```

Comment:
In custom ticket groups:
- Open a smart group;
- Click on a filter and edit the title;
- Click on "update filter";
- Click on that filter again

The "update filter" button is enabled but it shouldn't

P.S. Something analogous happens with v4 groups (same steps), but that's already in staging, so not sure if you want to/can fix it here